orchestration/main.go

```diff
@@ -83,10 +85,16 @@ func main() {
 			LocalFolder: paths.host.source,
 			MetaFolder:  paths.host.meta,
 			KeyFile:     paths.host.key,
+			Update: reuseScmCheckout,
 			Env:         env,
 		},
 	}
-	if err := f.Fetch(containerLogger); err != nil {
```

Comment:
The last bullet point in the PR description: if an update fails, re-launch the scm container in checkout mode to do a fresh checkout.